cmd/krew/cmd/internal/setup_check.go

```diff
@@ -58,7 +59,11 @@ func IsBinDirInPATH(paths environment.Paths) bool {
 
 	binPath := paths.BinPath()
 	for _, dirInPATH := range filepath.SplitList(os.Getenv("PATH")) {
-		if dirInPATH == binPath {
+		normalizedDirInPATH, err := filepath.Abs(dirInPATH)
+		if err != nil {
+			panic(errors.Wrapf(err, "cannot get absolute path for PATH part %s", dirInPATH))
```

Comment:
Let's not panic here. This is a convenience check and maybe `krew` would just work fine, even though we are not able to resolve the path. So maybe log the result at warning level and `continue`?